### ISSUE TITLE
BTable keyboard access

### ISSUE THREAD

**Issue by adholland** [bootstrap-vue-next/bootstrap-vue-next#1259]
We need to be able to have the same Bootstrap-vue keyboard navigation for the table components that they have. We currently lack this functionality. We need to add support where Bootstrap-vue has
